File: medstask/exponential/exp.py
def exponential(x,n):
    result=1.0
    term=1
    for i in range (1,n+1):
       term *= x / i
       result += term
    return result
x=float(input("Enter Value of x:"))
n=int(input("Enter Value of terms:"))
result = exponential(x, n)
print(f"Approximation of e^{x} using {n} terms: {result:.10f}")

File: medstask/exponential/exp.c
#include <stdio.h>
double exponential( double x, int n){
    double result=1.0;
    double term=1.0;
    
    for (int i=1 ; i<=n; i++){
        term*=x/i;
        result+=term;
    }
    return result;

}
int main() {
    double x;
    int terms;

    printf("Enter value of x: ");
    scanf("%lf", &x);

    printf("Enter number of terms: ");
    scanf("%d", &terms);

    double result = exponential(x, terms);
    printf("Approximation of e^%.2f using %d terms: %.10f\n", x, terms, result);

    return 0;
}